TestAutomationCourse/src/md/tekwill/homework/lesson6/inheritance/SideCarMotorcycle.java

```diff
@@ -0,0 +1,17 @@
+package md.tekwill.homework.lesson6.inheritance;
+
+public class SideCarMotorcycle extends Vehicle {
```

Comment:
In context of inheritance, consider adding constructors in child and parent classes that result in a child constructor calling parent constructor by super() keywork, although sometimes you may also need to use it as you did :)